test/end_to_end/test_create_runs.py

```diff
@@ -53,6 +53,7 @@ def test_create_run_from_mesh_outcrop_explicit_files(tmp_path, end_to_end_fixtur
     create_run_from_mesh(
         mesh_directory=mesh_directory,
         target_directory=new_directory,
+        append_zones=['top'],
```

Comment:
This is an unrelated change, but I added it to ensure the append feature was tested somewhere.